### ISSUE TITLE
iCloud Documents folder should be home (if iCloud enabled).

### ISSUE THREAD

**Issue by louisdh** [louisdh/terminal#14]
Currently home is set to the app's local folder (which has its local Documents folder as a subfolder). When iCloud is enabled, executing `cd ~` should bring the user to the iCloud documents folder.

This can probably be fixed with a `setenv`, though my first attempts have been futile. 
Not sure why this doesn't work:
```swift
setenv("HOME", DocumentManager.shared.activeDocumentsFolderURL.path.utf8CString, 0)
```

@holzschu Any clue what I'm doing wrong?

**Comment by holzschu** [comment]
"HOME" is already defined by the system, so you probably need to set the 3rd parameter to "1" (overwrite). 

But HOME might be used by other parts of the system (they defined it, probably because they use it). Also, the user might want to go to the Application local folder, or move things from the home Documents folder to the iCloud documents folder. I would be more in favor of defining a different variable ("CLOUDHOME"?) and changing `parseArgument()` so `~` expands to "CLOUDHOME" if it is defined, and to "HOME" otherwise.

**Comment by holzschu** [comment]
Question: where should we place configuration files? in $HOME or in the iCloud directory? Probably the latter. I'm talking about cacert.pem file, SSH keys, .curlrc file... 

If it is in the iCloud directory, I should update the `initializeEnvironment()` function.

**Comment by louisdh** [comment]
Yeah, if iCloud is enabled then it probably makes sense to have the configuration files in the iCloud directory.
